app/assets/javascripts/modules/moj.AgeLabel.js

```diff
@@ -0,0 +1,83 @@
+// Age labels for MOJ
+// Dependancies: moj, jQuery
+
+(function () {
+
+  'use strict';
+
+  var AgeLabel = function($el, options) {
+    this.init($el, options);
+    return this;
+  };
+
+  AgeLabel.prototype = {
+
+    defaults: {
+      threshold: 18,
+      labelUnder: 'Child',
+      labelOver: 'Adult'
+    },
+
+    init: function($el, options) {
+      this.settings = $.extend({}, this.defaults, options);
+      this.cacheEls($el);
+      this.bindEvents();
+      this.updateLabel();
+    },
+
+    cacheEls: function($el) {
+      this.$date = $el.find('.AgeLabel-date');
+      this.$label = $el.find('.AgeLabel-label');
+    },
+
+    bindEvents: function() {
+      this.$date.on('change', 'input', $.proxy(this.updateLabel, this));
+    },
+
+    isValidDate: function() {
```

Comment:
This is ambiguously named: `isValidDate` implies it returns true/false when, in fact, it's returning null/date. I'd probably just rename it `getDate` or something?